scss/bootstrap/overrides/style/_navs.scss

```diff
@@ -55,6 +55,18 @@
   }
 }
 
+.nav-pills {
+  margin-bottom: 15px;
+
+  > li {
+    // Active state
+    &.active > a {
```

Comment:
The selector which was causing the blue bg was
`#bootstrap-theme .nav-pills>li.active>a:focus` that is why only applied for active items.
Also checked in the UI that non-active items does not have this bg issue.